src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -127,6 +127,7 @@ public static function defaultSettings() {
       'manifesturl_source' => 'iiifmanifest',
       'max_width' => 720,
       'max_height' => 480,
+      'ia_reader_images_base_url' => '',
```

Comment:
Formatters using the ::getSetting method will eventually call this one here

see
```public function getSetting($key) {
    // Merge defaults if we have no value for the key.
    if (!$this->defaultSettingsMerged && !array_key_exists($key, $this->settings)) {
      $this->mergeDefaults();
    }
    return $this->settings[$key] ?? NULL;
  }
```
Which means by setting the return of getIABookReaderImagesDefaultBaseURL() directly you can save yourself that function and you can also call the defaultSettings method directly https://github.com/esmero/format_strawberryfield/pull/420/files#diff-b23e313798ac690e86b6697e23a4b490f091ea3b026b9480def3634656daf722R217 there instead of the private function

Reply:
I added the default url to the format_strawberryfield.schema.yml but I wasn't able to reuse or access it as the default value in the defaultSettings() function so I hardcoded the url there again.